Hunting Queries/BehaviorAnalytics/Anomalous RDP Activity.yaml

```diff
@@ -15,5 +15,5 @@ query: |
  BehaviorAnalytics
  | where ActivityType == "LogOn"
  | where ActionType == "RemoteInteractiveLogon"
- | where ActivityInsights.FirstTimeUserLoggedOnToDevice == true
+ | where ActivityInsights contains "True"
```

Comment:
It is possible to use "has" here instead of "contains".  It is much more efficient.

Reply:
done